group_vars/matrix_servers

```diff
@@ -1236,10 +1236,6 @@ matrix_mautrix_wsproxy_systemd_required_services_list: |
 
 matrix_mautrix_wsproxy_homeserver_domain: "{{ matrix_domain }}"
 
-matrix_mautrix_wsproxy_homeserver_address: "{{ matrix_homeserver_container_url }}"
-matrix_mautrix_wsproxy_hostname: "{{ matrix_server_fqn_matrix }}"
-matrix_mautrix_wsproxy_path_prefix: /_matrix/wsproxy
```

Comment:
I have my current setup working with nginx on this path, so I still need that to work. I think we just need to get the Path right.